### PR TITLE
Factor out the common get-var prefix into a constant

### PR DESCRIPTION
**Pull request by nabsul**

### CODE REVIEW

File: includes/class-wc-siftscience-admin.php [hunk 1]
```diff
@@ -22,13 +22,15 @@
...
-		private const GET_VAR_CLEAR_LOGS  = 'wc_sift_clear_logs';
 		private const NONCE_ACTION_PREFIX = 'nonce_action_';
 
+		private const GET_VAR_PREFIX      = 'wc_sift_';
+		private const GET_VAR_SET_STATS   = self::GET_VAR_PREFIX . 'send_stats';
+		private const GET_VAR_CLEAR_STATS = self::GET_VAR_PREFIX . 'clear_stats';
+		private const GET_VAR_RESET_GUID  = self::GET_VAR_PREFIX . 'reset_guid';
+		private const GET_VAR_TEST_SSL    = self::GET_VAR_PREFIX . 'test_ssl';
+		private const GET_VAR_CLEAR_LOGS  = self::GET_VAR_PREFIX . 'clear_logs';
+
```
Comment (ramico) [inline includes/class-wc-siftscience-admin.php:33]: the prefix is not a part of the get var name this couldn't be the case when this was done in options file they all had a schema 

the concat is done to differentiate nonces var names from get var name

**Review by ramico — CHANGES_REQUESTED** [review]

File: includes/class-wc-siftscience-admin.php (hunk 1 above)
Comment (ramico) [reply includes/class-wc-siftscience-admin.php:33]: ps method get action name was not deleted

File: includes/class-wc-siftscience-admin.php [hunk 2]
```diff
@@ -22,13 +22,15 @@
 	class WC_SiftScience_Admin {
 		private const ADMIN_ID            = 'siftsci';
 		private const ADMIN_LABEL         = 'Sift';
-		private const GET_VAR_SET_STATS   = 'wc_sift_send_stats';
-		private const GET_VAR_CLEAR_STATS = 'wc_sift_clear_stats';
-		private const GET_VAR_RESET_GUID  = 'wc_sift_reset_guid';
-		private const GET_VAR_TEST_SSL    = 'wc_sift_test_ssl';
-		private const GET_VAR_CLEAR_LOGS  = 'wc_sift_clear_logs';
 		private const NONCE_ACTION_PREFIX = 'nonce_action_';
```
[the remark this answers is not in the document]
Comment (nabsul) [reply includes/class-wc-siftscience-admin.php:25]: I'll call it `nonce_`. `wc_` is redundant.

File: includes/class-wc-siftscience-admin.php (hunk 1 above)
Comment (nabsul) [reply includes/class-wc-siftscience-admin.php:33]: I don't understand.

Comment (nabsul) [reply includes/class-wc-siftscience-admin.php:32]: This needs testing, but we could simplify and use the same value in nonce get-var and nonce-action.

**Review by ramico — APPROVED** [review]

**Pull request merged**